jupyter-leaflet.json

```diff
@@ -1,5 +1,5 @@
 {
-    "load_extensions": {
-        "jupyter-leaflet/extension": true
-    }
+  "load_extensions": {
+    "jupyter_leaflet/extension": true
```

Comment:
I would say this is a risky change, this will trip people up that don't see it at first glance, if possible I'd keep it as is.